nemo/collections/asr/parts/submodules/ctc_decoding.py

```diff
@@ -98,6 +98,10 @@ class AbstractCTCDecoding(ConfidenceMixin):
                     Which aggregation type to use for collapsing per-token confidence into per-word confidence.
                     Valid options are `mean`, `min`, `max`, `prod`.
 
+                tdt_include_duration: Bool flag indicating that the duration confidence scores are to be calculated and
```

Comment:
Why should it be in the CTC config (here and below)? I think in CTC models we do not need TDT confidence params

Reply:
To my understanding, comments in ctc_decoding and rnnt_decoding just display the content of the confidence config.